BUILDING.md

```diff
@@ -253,6 +253,23 @@ To install this version of Node.js into a system directory:
 $ [sudo] make install
 ```
 
+#### Building a debug build
+
+If you run into a issue where Node.js segfaults and the "segfault-handler" does not provide enough information, you might have to build a debug build of Node.js to get enough information to debug the issue further, this is done by :
+
+```console
+$ ./configure --debug
+$ make -j4
+```
+
+A make with "./configure --debug" generates two binaries, the regular release one in "out/Release/node" and a debug build in "out/Debug/node", only the release version is actually installed when you run "make install". 
```

Comment:
I'm not sure but I think this 'A' is not needed here, so it's just 'Make with ...' or '`make` with'.

Also, I think it'll be better with 'a debug build in' -> 'a debug binary in'.

Reply:
Also maybe it is worth to replace double quotes with backticks for all 4 cases here.